### PR TITLE
Support filling Roxygen blocks that contain backtick characters

### PR DESCRIPTION
**Pull request by nfdisco**
I've run the tests and it didn't seem to break anything.

### CODE REVIEW

File: lisp/ess-roxy.el [hunk 1]
```diff
@@ -731,19 +732,22 @@ list of strings."
...
-         (temp-table (make-syntax-table S-syntax-table))
          (paragraph-start (concat "\\(" ess-roxy-re "\\(" paragraph-start
-                                  "\\|[ \t]*@" "\\)" "\\)\\|\\(" paragraph-start "\\)")))
-     ;; Prevent the roxy prefix to be interpreted as comment or string
-     ;; starter
-     (modify-syntax-entry ?# "w" temp-table)
-     (modify-syntax-entry ?' "w" temp-table)
+                                  "\\|[ \t]*@" "\\)" "\\)\\|\\(" paragraph-start "\\)"))
+         (temp-table (if ,examples
+			 (make-syntax-table S-syntax-table)
```
Comment (lionel-) [inline lisp/ess-roxy.el]: Could you use spaces instead of tabs please.

File: test/literate/roxy.R [hunk 2]
```diff
@@ -74,3 +74,24 @@ NULL
 
 ##'
 ##' ¶
+
+
+### 3 ----------------------------------------------------------------
```
Comment (lionel-) [inline test/literate/roxy.R]: This test case could be a bit more minimal, i.e. just

```r
##' ¶Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor `incididunt' ut labore et ``dolore'' magna aliqua.
```

Without any other field so it's clearer we are testing for quotes (and we probably should come up with a way of labelling the test cases).

**Comment by lionel-** [comment]
Otherwise looks good!

**Comment by nfdisco** [comment]
Done!

Apparently the test needs to be terminated with a NULL otherwise it fails. In a buffer it works with or without the NULL.

**Comment by lionel-** [comment]
Thanks!

**Pull request merged**